vcs/rsync.go

```diff
@@ -0,0 +1,45 @@
+package vcs
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"path/filepath"
+)
+
+func init() {
+	Register(newRsync, "rsync")
+}
+
+type RsyncDriver struct{}
+
+func newRsync(b []byte) (Driver, error) {
+	return &RsyncDriver{}, nil
+}
+
+func (g *RsyncDriver) HeadRev(dir string) (string, error) {
+	return "n/a", nil
```

Comment:
The way this is setup, Hound will continue to try to poll this vcs even though it can never update. I think it would be better to return something sensible for the revision so that pulling works as expected. That said, I do think that an rsync vcs should not poll by default. It should require an explicit push update to tell the server to go fetch the new version. The server supports push updates, but there isn't a convenient way to trigger one right now.